tests/pathfinding/test_capacity_updates.py

```diff
@@ -0,0 +1,506 @@
+"""
+The tests in this module mock events creation by using a mock blockchain listener.
+
+The Capacity Updates should be tested in various states
+"""
+
+from typing import List
+
+import pytest
+from eth_utils import decode_hex
+
+from pathfinding_service import PathfindingService
+from pathfinding_service.exceptions import MessageException
+from pathfinding_service.model import TokenNetwork
+from raiden.messages import UpdatePFS
+from raiden.utils import CanonicalIdentifier
+from raiden.utils.typing import ChainID, ChannelID, Nonce, TokenAmount, TokenNetworkAddress
+from raiden_libs.types import Address, ChannelIdentifier
+
+
+DEFAULT_TOKEN_NETWORK_ADDRESS = Address('0x6e46B62a245D9EE7758B8DdCCDD1B85fF56B9Bc9')
+DEFAULT_TOKEN_NETWORK_ADDRESS_BYTES = TokenNetworkAddress(
+    decode_hex(DEFAULT_TOKEN_NETWORK_ADDRESS),
+)
+DEFAULT_TOKEN_ADDRESS = Address('0x44Ac22fd9672cC559Ab171603D474cEA8a2D7b4D')
+
+"""
+def get_UpdatePFS_message(
```

Comment:
Do you still need this, otherwise please remove.

Reply:
done